packages/manager/apps/pci-kubernetes/src/pages/detail/log/Streams.page.tsx

```diff
@@ -97,7 +103,12 @@ export default function StreamsPage() {
         color={ODS_THEME_COLOR_INTENT.primary}
         size={ODS_BUTTON_SIZE.sm}
         variant={ODS_BUTTON_VARIANT.stroked}
-        onClick={gotoAddDataStream}
+        onClick={() => {
+          gotoAddDataStream();
```

Comment:
If this is a redirection are there chances that the redirection stops what come next?

Reply:
yes, swapped the calls